api/base/serializers.py

```diff
@@ -279,6 +299,53 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
+        instance_mapping = {item._id: item for item in instance}
```

Comment:
Will take a look. 

To be sure I understand the idea: Will the serializer `Meta.unique_identifier` then need to be the same value as `IDField.source`? If so, this still seems to be an issue with separation of concerns: the same rule (name of id field) now has to live in two places simultaneously. This will probably *work*, but introducing the same rules / code in multiple places can be fragile and harder to maintain.